src/gui/optionsdialog.ui

```diff
@@ -2442,10 +2442,7 @@ Disable encryption: Only connect to peers without protocol encryption</string>
                   <number>-1</number>
                  </property>
                  <property name="maximum">
-                  <number>999</number>
-                 </property>
-                 <property name="value">
-                  <number>3</number>
+                  <number>2147483647</number>
```

Comment:
What is default value if you don't specify anything?

Reply:
99 according to https://doc.qt.io/qt-5/qspinbox.html#maximum-prop